packages/fast-components-styles-msft/src/dialog/index.ts

```diff
@@ -36,7 +34,7 @@ const styles: ComponentStyles<IDialogClassNameContract, IDesignSystem> = (config
             top: "50%",
             transform: "translate(-50%, -50%)",
             background: backgroundColor,
-            boxShadow: dialogShadow
+            ...elevation("e14", foregroundColor)(designSystem)
```

Comment:
given how we are using this, should it be an enum? strings seem breakable.

Reply:
I don't think an enum allows us to be extensible and support other people creating their own ramp.

Reply:
are we allowing people to create their own ramp within MSFT?

Reply:
Different teams may require different values. We do this with the typeramp and it follows our principles of extensibility. We give good defaults, but they can change or manipulate to serve their requirements; I think the same should be true here.

Reply:
I think we need to find a solution that allows us to have type safety within our own config, if you find a good solution we should also apply this to typography.

Reply:
Typography will be another task. Can you create one for it?

Reply:
@janechu I think we should explore if it will work for typography, but while these were similar, they are quite different in terms of the functional requirements. 

I *do* think we can make an enum change in typography, just saying that I don't think the change will be 1:1